core-tests/src/test/java/org/mule/runtime/core/internal/processor/strategy/ProactorStreamEmitterProcessingStrategyTestCase.java

```diff
@@ -315,7 +315,6 @@ public void singleCpuLightConcurrentMaxConcurrency1() throws Exception {
                                                                                                     1, true)),
                        true, CPU_LITE, 1);
     assertThat(threads, hasSize(1));
-    assertThat(threads, not(hasItem(startsWith(CPU_LIGHT))));
```

Comment:
if this is expected now, just remove the `not`

Reply:
or better yet, count the occurrences as `ProactorStreamWorkQueueProcessingStrategyTestCase` does with the ring buffer threads